ros/src/tl_detector/light_classification/tl_classifier.py

```diff
@@ -73,23 +76,42 @@ def get_classification(self, image):
         mask_yel = cv2.inRange(hsv, self.lower_yel, self.upper_yel)
         mask_gre = cv2.inRange(hsv, self.lower_gre, self.upper_gre)
 
+        # Debug storage
```

Comment:
Think about adding a debug flag here or add a debug function and then only uncomment the call to it - given that you think this code could still be useful (otherwise just remove it).

Reply:
Done